requirements-testing.txt

```diff
@@ -1,15 +1,15 @@
 coverage[toml] ~= 7.5
 coverage-conditional-plugin == 0.9.*
-deadline-cloud-test-fixtures == 0.6.*
 pytest ~= 8.2
 pytest-cov == 5.0.*
+/home/moorec/Projects/git/deadline-cloud-test-fixtures/dist/deadline_cloud_test_fixtures-0.6.2.post18+g7526d2c.tar.gz
```

Comment:
typo

Reply:
This will be updated once https://github.com/aws-deadline/deadline-cloud-test-fixtures/pull/115 is merged.